src/main/kotlin/apply/domain/evaluationtarget/EvaluationTargetRepository.kt

```diff
@@ -3,6 +3,8 @@ package apply.domain.evaluationtarget
 import org.springframework.data.jpa.repository.JpaRepository
 
```

Comment:
```suggestion

fun EvaluationTargetRepository.findEvaluationTargetOf(evaluationId: Long, userId: Long): EvaluationTarget {
    return findByEvaluationIdAndUserId(evaluationId, userId) ?: throw IllegalArgumentException("평가 대상자가 아닙니다.")
}

```

AssignmentService 클래스에 `findEvaluationTargetOf` 메서드를 만드는 대신에 Repository에서 확장함수를 만들어보면 어떨까요? 😃

Reply:
repository.findByEvaluationIdAndUserId(evaluationId, userId)의 검색 결과가 없으면 "평가 대상자가 아닙니다." 메시지로 예외 상황을 통일해도 괜찮을까요?

기존 코드에 이런 식으로 다중 조건을 갖는 조회를 Repository 확장 함수로 뺀 케이스가 없어 조심스럽네요. 
답변 듣고 반영하겠습니다.